spring-ai-spring-boot-autoconfigure/src/test/java/org/springframework/ai/autoconfigure/vectorstore/azure/AzureVectorStoreAutoConfigurationIT.java

```diff
@@ -81,8 +82,8 @@ public static void beforeAll() {
 	public void addAndSearchTest() {
 
 		contextRunner
-			.withPropertyValues("spring.ai.vectorstore.azure.indexName=my_test_index",
-					"spring.ai.vectorstore.azure.defaultTopK=6",
+			.withPropertyValues("spring.ai.vectorstore.azure.initializeSchema=true",
```

Comment:
this test passes when `initializeSchema=false`.  I would expect it to fail.

Reply:
As we discussed, this happens because the index is already on the Azure vector store and is never deleted.